### ISSUE TITLE
request: android support

### ISSUE THREAD

**Issue by gkovacs** [birchill/10ten-ja-reader#55]
You can probably look at zhongwen https://github.com/leezu/zhongwen/ https://addons.mozilla.org/en-US/firefox/addon/zhongwen-chinese-english/ for pointers on implementation - it is an analogous extension that does support the android version of firefox

**Comment by birtles** [comment]
Thanks! How do you show the pop up on a touch device? By long-pressing on the word?

**Comment by gkovacs** [comment]
You just tap the word

**Comment by birtles** [comment]
Ah interesting. Do you need to enable the add-on from the menu first? Or is it always on?

**Comment by gkovacs** [comment]
Yes you enable it from the add on menu first. You can just try the
extension itself if you want details it's open source
https://github.com/leezu/zhongwen/ and on the addons store
https://addons.mozilla.org/en-US/firefox/addon/zhongwen-chinese-english/

Brian Birtles <notifications@github.com> 于 2018年9月30日周日 23:00写道：

**Comment by nicolasmaia** [comment]
Would love to see this!

**Comment by birtles** [comment]
Apparently this works already on Firefox Nightly:

https://twitter.com/boardsmorgas/status/1384025797275447296

See also:

https://twitter.com/boardsmorgas/status/1384026419156590597

There probably is some work to be done with regards to selection handling, however:

> i think the only thing that can improve it is forcing it to perform a dictionary search on the highlighted text instead of on whatever character my fat finger touches first 
😉
[(Source)](https://twitter.com/boardsmorgas/status/1384317234911354880)

**Comment by jonathan-s** [comment]
Works fine for android for me, so I think this can be closed.
